projects/product-configuration/.release-it.json

```diff
@@ -0,0 +1,34 @@
+{
+  "git": {
+    "requireCleanWorkingDir": true,
+    "requireUpstream": false,
+    "tagName": "product-configuration-${version}",
+    "commitMessage": "Bumping product-configuration version to ${version}",
+    "tagAnnotation": "Bumping product-configuration version to ${version}"
+  },
+  "npm": {
+    "publishPath": "./../../dist/product-configuration"
+  },
+  "hooks": {
+    "after:version:bump": "cd ../.. && yarn build:core:lib:product-configuration"
```

Comment:
Let's use a shorter name than `product-configuration`. Also, make sure that this matches the script in `package.json`.